src/components/carousel/_index.scss

```diff
@@ -175,6 +175,12 @@
 			&__image-counter-label {
 				@include scss.component-properties("carousel-fullscreen-image-counter");
 			}
+
```

Comment:
I noticed an issue while reviewing this, it's not related to your changes but we should clean it up here. Above in the `&__actions` rule there's CSS setting `display: flex`. Since that property is also set on the token, the final CSS has two `display` properties like so:
```
.c-carousel__actions {
    display: flex;
    ...
    display: var(--c-carousel-actions-display);
    ...
}
```

The 'hardcoded' `display` prop should be removed from this file.

Reply:
@vgalatro thank you, the change is done.